### PR TITLE
Martyrdoms - new and additions

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/LIT6723Yohanni.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6723Yohanni" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Martyrdom of Yoḥanni</title>
```
Comment (MarcinKrawczuk) [inline new/LIT6723Yohanni.xml:8]: I think this record is not needed and we should use https://betamasaheft.eu/works/LIT4051Homily/.

File: 2001-3000/LIT2085OntheL.xml [hunk 2]
```diff
@@ -63,6 +78,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:CLP46i_may_form_part_of" active="LIT2085OntheL" passive="LIT1493Gadlas"/>
                <relation name="saws:isAttributedToAuthor" active="LIT2085OntheL" passive="PRS5720JohnChr"/>
```
Comment (MarcinKrawczuk) [inline 2001-3000/LIT2085OntheL.xml:80]: Perhaps also mark as different from LIT2093OnHero, also let me repeat my suggestion to use Proverbio's book as clavis for Chrysostomian homilies

File: 3001-4000/LIT3164JohnDailami.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">ስምዕ፡ ዘዮሐንስ፡ ሰይላሚ፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">Sǝmʿ za-Yoḥannǝs Daylāmi</title>
```
Comment (MarcinKrawczuk) [inline 3001-4000/LIT3164JohnDailami.xml:8]: ደይላሚ

File: 6001-7000/LIT6173JamesIn.xml [hunk 4]
```diff
@@ -6,8 +6,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
                 <title xml:lang="en" xml:id="t1">Martyrdom of James Intercisus</title>
-                <title xml:lang="gez" xml:id="t2">ስምዕ፡ ዘቅዱስ፡ ግሙድ፡ መስተጋድል፡ ያዕቆብ፡ ዘረከቦ፡ ዘተፈጸመ፡
-                ስምዑ፡ በምድረ፡ ፋርስ፡ በሀገረ፡ እንተ፡ ስማ፡ በልቃ፡ በመዋዕለ፡ ሰርደ፡ ንጉሥ፡</title>
+                <title xml:lang="gez" xml:id="t2">ስምዕ፡ ዘቅዱስ፡ ግሙድ፡ መስተጋድል፡ ያዕቆብ፡ ዘረከቦ፡ ዘተፈጸመ፡ ስምዑ፡ በምድረ፡ ፋርስ፡ በሀገረ፡ እንተ፡ ስማ፡ በልቃ፡ በመዋዕለ፡ ሰርደ፡ ንጉሥ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Sǝmʿ za-qǝddus gǝmmud mastagādl Yāʿqob 
+                    za-rakabo za-tafaṣṣama sǝmʿu ba-mǝdra Fārs ba-hagara ʾǝnta sǝmā Balqā ba-mawāʿla sarda nǝguś</title>
```
Comment (MarcinKrawczuk) [inline 6001-7000/LIT6173JamesIn.xml:11]: mawāʿǝla and Sarda?

**Review by MarcinKrawczuk — COMMENTED** [review]
Just a few comments here and there

File: new/LIT6723Yohanni.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6723Yohanni.xml:8]: LIT1521 and LIT4051 are on Yoḥanni of Dabra ʿAśā, of Tigray, surely not a martyr (Gadl YDA in BN Éth. 132, BL Add. 16,230, EMML nos. 1734, 3445, 4268, etc, this martyrdom is only in EMML 7602)
(I am not sure who BHO 496 is about... there may be an error in LIT1521)

File: 2001-3000/LIT2085OntheL.xml (hunk 2 above)
Comment (eu-genia) [reply 2001-3000/LIT2085OntheL.xml:80]: I will get back to this, thank you

File: new/LIT6723Yohanni.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT6723Yohanni.xml:8]: BHO 486 is also Y of DA, all correct

**Review by MarcinKrawczuk — APPROVED** [review]

**Pull request merged**